### ISSUE TITLE
Feature request : Play track only once (no loop)

### ISSUE THREAD

**Issue by GerhardSchreurs** [schismtracker/schismtracker#194]
Hi there, maybe I'm totally missing something, but is there any way to play a track only once?
Pressing F5, Ctrl+F5, Shift+F6, F7 (et cetera) loops a track.

If such a feature exists, please let me know. If not, I'd like to have it implemented.

Suggestion: Use another key combo like Ctrl + Shift + F5 to play a track once.

**Comment by jangler** [comment]
No, I don't believe there's any way to do that. IT doesn't have a pattern command to end the song either.

**Comment by guaka** [comment]
`W01` was aworkaround I've used a lot, e.g.
![image](https://user-images.githubusercontent.com/54728/59445874-4d4fec00-8df8-11e9-8cff-234cd7c8a753.png)

Then when it loops back to the beginning the global volume is 0.

**Comment by GerhardSchreurs** [comment]
@guaka The workaround is nice suggestion, thanks.
@jangler Yeah, I do believe that there wasn't such a feature in the old Impulse Tracker either. Still, it would be nice if Schism could do this.

**Comment by strongholdmedia** [comment]
We used to set tempo to some outlandish values (like AFF/T20 or so) so that quite some time passed before it started again.

_Off: is the guy who made the track in the screenshot using the actual impulse tracker pro audio spectrum driver as a sample? :D We used exes and jpgs for pseudo-white noise but I find this "ouroboros solution" hilarious. :D_
